libs/utils/utils.js

```diff
@@ -690,7 +690,8 @@ async function decorateIcons(area, config) {
   if (icons.length === 0) return;
   const { miloLibs, codeRoot } = config;
   const base = miloLibs || codeRoot;
-  await new Promise((resolve) => { loadStyle(`${base}/features/icons/icons.css`, resolve); });
+  loadStyle(`${base}/features/icons/icons.css`);
+  loadLink(`${base}/img/icons/icons.svg`, { rel: 'preload', as: 'fetch', crossorigin: 'anonymous' });
   const { default: loadIcons } = await import('../features/icons/icons.js');
   await loadIcons(icons, config);
```

Comment:
I would suggest one of two things:

- Directly load the icons.svg if it's LCP relevant `await Promise.all([css, icons, js])`

OR

- move the css and svg loading into icons.js wait for both of those resources to be loaded in parallel then execute the logic.  In this case you could do this:
```js
async function decorateIcons(area, config) {
  const icons = area.querySelectorAll('span.icon');
  if (icons.length === 0) return null;
  return import('../features/icons/icons.js')
    .then(({ default: loadIcons }) => loadIcons(area, config));
}
```

or just load the utils from the icons.js

Reply:
"Preloading is best suited for resources typically discovered late by the browser."
-- https://web.dev/articles/preload-critical-assets